design-system/packages/fields/src/FieldDescription.tsx

```diff
@@ -0,0 +1,27 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+
+import { jsx, useTheme } from '@keystone-ui/core';
+
+type FieldDescriptionProps = {
+  id: string;
+  children: string | null;
+};
+
+export const FieldDescription = (props: FieldDescriptionProps) => {
+  const { spacing } = useTheme();
+  if (props.children === null) {
+    return null;
+  }
+  return (
+    <span
+      css={{
+        color: '#667092',
```

Comment:
```suggestion
        color: '#6b7280',
```
sourced from:

https://github.com/keystonejs/keystone/blob/73f4d566dd580b91a096e101327786df41169347/design-system/packages/core/src/themes/default.ts#L54-L71

The orginal color is from the new UI color tokens which we should avoid using for the time being